.ci/ansible_install.py

```diff
@@ -11,7 +11,8 @@
         'pip install '
             '-r tests/requirements.txt '
             '-r tests/ansible/requirements.txt',
-        'pip install -q ansible=={0}'.format(ci_lib.ANSIBLE_VERSION)
+        # encoding is required for installing ansible 2.10 with pip2, otherwise we get a UnicodeDecode error
+        'LC_CTYPE=en_US.UTF-8 LANG=en_US.UTF-8 pip install -q ansible=={0}'.format(ci_lib.ANSIBLE_VERSION)
```

Comment:
I also encountered this, pinning to `pip<20.2` also avoids it https://github.com/dw/mitogen/pull/762/commits/5ecf6488d4dcacf0ab65105b0e19e400f96e7c4f